config/mtl.toml

```diff
@@ -192,7 +192,7 @@ count = 15
 	instance_count = "40"
 	domain_types = "0"
 	load_type = "0"
-	module_type = "5"
+	module_type = "4"
```

Comment:
@plbossart fyi - looks like we still need some module type IDs for IPC4 compat but I dont think they are needed by the driver for IPC4 ?

Reply:
we don't use module type. We set the info in topology, such as uuid and dapm type like dai_in dai_out